renormalizer/mps/gs.py

```diff
@@ -556,4 +556,7 @@ def hop(x):
     else:
         assert False
     logger.debug(f"use {algo}, HC hops: {count}")
-    return e, c
+    if nroots == 1:
+        return e, c/np.sign(np.max(c))
+    else:
+        return e, c/np.sign(np.max(c, axis=0))
```

Comment:
This might cause the bug in CI. Taking max along the 0th axis will eliminate the 0th axis, resulting a vector with size N, where N is the dimension of c.